src/libsyncengine/propagation/executor/executorworker.cpp

```diff
@@ -2301,8 +2301,7 @@ ExitInfo ExecutorWorker::propagateDeleteToDbAndTree(SyncOpPtr syncOp) {
     }
 
     if (!targetUpdateTree(syncOp)->deleteNode(syncOp->correspondingNode())) {
-        LOGW_SYNCPAL_WARN(_logger,
-                          L"Error in UpdateTree::deleteNode: node name=" << SyncName2WStr(syncOp->correspondingNode()->name()));
+        logCorrespondingNodeErrorMsg(syncOp);
```

Comment:
It seems manageable to write a short unit test on the method `ExecutorWorker::propagateDeleteToDbAndTree` (We may mock `deleteFromDb`).

Reply:
Will do that later.